src/uis/polkadot_js.rs

```diff
@@ -1410,6 +1410,11 @@ impl ContractsUi for crate::uis::Ui {
 
         log::info!("[{}] entering contract address {:?}", log_id, contract_addr);
         let path = "//input[@data-testid = 'contract address']";
+
+	// To overcome a UI quirk with input, we type the address here in two batches.
```

Comment:
Can you add more explanation. I don't understand what the issue is from this comment.

```suggestion
	// To overcome a UI quirk with input, we type the address here in two batches.
```